virttest/libvirt_xml/vm_xml.py

```diff
@@ -548,6 +548,20 @@ def set_controller(self, controller_list):
             devices_element.append(element.getroot())
         self.xmltreefile.write()
 
+    def get_controller(self, controller_type=None, model=None):
```

Comment:
we can modify existing `get_controllers()` method in the same file, rather than a new method

Reply:
Oh, I didn't notice that method before. I will check, thanks very much.